docs/src/install.md

```diff
@@ -58,6 +58,8 @@ After the initial installation, you need to update your channels once, to be abl
 sudo nix-channel --update
 ```
 
+Don't forget to set a password for your user (default is `nixos` and is in the `wheel` group) in case you'll set ` security.sudo.wheelNeedsPassword = true;`.
```

Comment:
```suggestion
Don't forget to set a password for your user (default is `nixos` and is in the `wheel` group) in case you'll set `security.sudo.wheelNeedsPassword = true;` which is the default.
```